src/legacy/core_plugins/interpreter/init.ts

```diff
@@ -25,9 +25,7 @@ import { register, registryFactory, Registry, Fn } from '@kbn/interpreter/common
 // @ts-ignore
 import { routes } from './server/routes';
 
-import { typeSpecs as types } from '../../../plugins/expressions/public';
-
-import { Type } from './common';
+import { typeSpecs as types, Type } from '../../../plugins/expressions/common';
```

Comment:
This is a server-side file, it *explicitly should not* import from the client side `public`. Because it used to import from `public` before, was a bug, and caused me spend much more time on this PR than it should have. The fact that it even worked before is pure luck.